pkg/git/github.go

```diff
@@ -154,7 +154,12 @@ func (g *GitHubGITProvider) GetStatus(ctx context.Context, sha string, group str
 	if err != nil {
 		return Status{}, err
 	}
-	log.Printf("Considering statuses %v\n", statuses)
+	var displays = make([]string, len(statuses))
+	for i := range statuses {
+		s := statuses[i]
+		displays = append(displays, fmt.Sprintf("%s: %s (%s)", *s.Context, *s.State, *s.Description))
```

Comment:
What is Context in this case?

Reply:
This is the ID for this status. Flux uses e.g. `kustomization/app-dev` (in AZDO, this is genre + name).